packages/features/bookings/UnconfirmedBookingBadge.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+
+import { trpc } from "@calcom/trpc/react";
+import Badge from "@calcom/ui/v2/core/Badge";
+
+export default function UnconfirmedBookingBadge() {
+  const { data: unconfirmedBookingCount } = trpc.useQuery(["viewer.bookingUnconfirmedCount"]);
+  if (!unconfirmedBookingCount) return null;
+  else
+    return (
+      <Link href="/bookings/unconfirmed">
+        <a>
+          <Badge rounded variant="orange">
+            {unconfirmedBookingCount}
+          </Badge>
+        </a>
+      </Link>
```

Comment:
Would be great to add a tooltip to show what's this number about, a "title" in the anchor should do.

Reply:
i think its clear once you click it, because it navigates to "unconfirmed" if you click the badge

Reply:
I'm just saying because it could also mean upcoming, both very useful numbers to have at hand

Reply:
Also, there's no UI/UX indication the badge has its own link, so I didn't noticed you could go to the unconfirmed page clicking on it. My suggestion would be to put a "title" to avoid complex tooltips for this that could read "Unconfirmed bookings" and add some `hover` state. All totally optional of course, just a suggestion.

Reply:
Added it, WDYT?
<kbd><img width="301" alt="image" src="https://user-images.githubusercontent.com/467258/194711452-1ef0dbc1-4255-47ee-b50c-d0f8b4ab30b0.png"></kbd> ➡️ <kbd><img width="301" alt="image" src="https://user-images.githubusercontent.com/467258/194711431-265ca292-b0f9-4cd2-934d-66983525359b.png"></kbd>

Reply:
<img width="1512" alt="CleanShot 2022-10-08 at 17 57 03@2x" src="https://user-images.githubusercontent.com/8019099/194718782-bd1e1778-ad20-4e19-b9cd-bde1bdb2e115.png">
wtf it shows up as -4 now on preview https://cal-pkq3e3781-cal.vercel.app/bookings/unconfirmed

pro@example.com
pro

Reply:
@leog

Reply:
I don't see it 🤔
<kbd><img width="1507" alt="image" src="https://user-images.githubusercontent.com/467258/194720384-879855d4-490f-45e2-b7d0-12f19a713ec2.png"></kbd>

Reply:
schedule a recurring event again. i cancelled them